index.js

```diff
@@ -8,12 +9,13 @@ const STATUS_COLORS = {
   info: 'green'
 };
 
+
 /**
  * Logger
  *
  * @param {object} winstonInstance
  */
-function logger(winstonInstance) {
+function logger(winstonInstance, {useColors=(tty.isatty(1) && process.env.NODE_ENV !== 'production')}={}) {
```

Comment:
I think that is easier to read a named constant instead of this, additionally I would like to support Node v4.x (it's a LTS) so if we can avoid default parameters would be great.